functionsHW9.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <iostream>
+#include <cctype>
```

Comment:
no includes in headers unless you really need them here for you functions prototypes